safe/test/utilities.py

```diff
@@ -69,7 +73,13 @@ def get_qgis_app():
     """
     global QGIS_APP, PARENT, IFACE, CANVAS  # pylint: disable=W0603
 
-    if iface:
+    from PyQt4.QtCore import QSettings
+    settings = QSettings()
+
+    current_locale = settings.value('locale/userLocale', 'en_US')
```

Comment:
Consider using our `set_general_setting` method. Be careful to change this in the unit test. It may break another test since you change the language to Indonesia in `test_qgis_app_locale`.

I suggest adding `QSettings` as a parameter so that you can specify which QSetting object that you want to change.

Reply:
thanks, updated